scripts/test.js

```diff
@@ -0,0 +1,20 @@
+// For any third party dependencies, like jQuery, place them in the lib folder.
```

Comment:
Thats the "main" entry for the test. We need to add one extra for the graph and start implementing the main app one.